packages/template-retail-react-app/app/components/reset-password/index.jsx

```diff
@@ -16,64 +16,99 @@ import ResetPasswordFields from '@salesforce/retail-react-app/app/components/for
 const ResetPasswordForm = ({submitForm, clickSignIn = noop, form}) => {
     return (
         <Fragment>
-            <Stack justify="center" align="center" spacing={8}>
-                <BrandLogo width="60px" height="auto" />
-                <Stack spacing={2}>
+            {!form.formState.isSubmitSuccessful && (
```

Comment:
Nit: Why don't we use if ternary here?

Reply:
Ah good point, the if ternary makes more sense here!